app/graphql/mutations/account/create.rb

```diff
@@ -0,0 +1,35 @@
+class Mutations::Account::Create < Mutations::Base
+  include PublicMutation
+  include RateLimitedMutation
+  include FancyMutation
+
+  description 'Create a new Kitsu account'
+
+  rate_limit do
+    limit 20, per: 1.hour
+    limit 20, per: 2.days
+  end
+
+  input do
+    argument :name, String,
+      required: true,
+      description: 'The name of the user'
+
+    argument :email, String,
+      required: true,
+      description: 'The email address to reset the password for'
+
+    argument :password, String,
+      required: true,
+      description: 'The password for the user'
+
+    argument :external_identity, Types::Input::Account::ExternalIdentity,
+      required: false,
+      description: 'An external identity to associate with the account on creation'
+  end
+  result Types::Account
+
+  def resolve(input:)
+    AccountMutator.create(**input)
```

Comment:
Just thinking a bit about this service object and are we gonna mix n match between doing class methods, and/or having these service object be something like `AccountCreator.new(**).call` or w/e.

Reply:
Class methods only. If you need to instantiate, just use a regular Service object